exdb/tests/integration_tests.py

```diff
@@ -546,12 +549,13 @@ def test_week_ahead(self):
                                               start_datetime=(now() + timedelta(days=2)),
                                               end_datetime=(now() + timedelta(days=3)),
                                               type=self.create_type(),
-                                              subtype=self.create_subtype(),
                                               goals="Test Goal",
                                               audience="b",
                                               status="ad",
                                               attendance=None,
                                               next_approver=self.clients['hs'].user_object)[0]
+        e2.subtypes.add(self.create_subtype())
+        e2.save()
```

Comment:
Calling `.save()` is unnecessary if you're only modifying a many-to-many. (And it also results in an extra DB query.)